scss/components/_accordion.scss

```diff
@@ -65,24 +67,30 @@ $accordion-content-padding: 1rem !default;
 }
 
 /// Adds styles for the title of an accordion item. Apply this to the link within an accordion item.
-@mixin accordion-title {
+@mixin accordion-title (
+  $padding: $accordion-item-padding,
+  $font-size: $accordion-title-font-size,
+  $color: $accordion-item-color,
+  $border: $accordion-content-border,
+  $background-hover: $accordion-item-background-hover
+) {
   display: block;
-  padding: $accordion-item-padding;
+  padding: $padding;
   line-height: 1;
-  font-size: $accordion-title-font-size;
-  color: $accordion-item-color;
+  font-size: $font-size;
+  color: $color;
```

Comment:
Why does some components have an alternative color, and others doesn't ?
It could be useful to use a common *color format* like a `($color-light, $color-dark)` list (for background too), and a mixin like `component-skin($background-list, $color-list, $pick-best-contrast)` to use it in components.